src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -53,6 +53,7 @@ namespace NuGetVSExtension
         Orientation = ToolWindowOrientation.Right)]
     [ProvideOptionPage(typeof(PackageSourceOptionsPage), "NuGet Package Manager", "Package Sources", 113, 114, true)]
     [ProvideOptionPage(typeof(GeneralOptionPage), "NuGet Package Manager", "General", 113, 115, true)]
+    [ProvideOptionPage(typeof(PackageSourceMappingOptionsPage), "NuGet Package Manager", "Package Source Mapping", 113, 116, true)]
```

Comment:
Please, add an entry in [NuGet.Tools\VSPackage.resx](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.Tools/VSPackage.resx) with the category name so it can be localized:

![image](https://user-images.githubusercontent.com/1192347/179121202-a395d862-2156-4b63-8084-09acb6bd69c4.png)

Name: 116
Value: Package Source Mapping

Reply:
Thank you. I didn't know I had to do this